core/src/test/java/com/novoda/accessibility/AccessibilityServicesTest.java

```diff
@@ -3,54 +3,68 @@
 import android.accessibilityservice.AccessibilityServiceInfo;
 import android.view.accessibility.AccessibilityManager;
 
-import java.util.Collections;
-import java.util.List;
-
-import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Mock;
 
+import java.util.Collections;
+
+import static android.accessibilityservice.AccessibilityServiceInfo.FEEDBACK_ALL_MASK;
+import static android.accessibilityservice.AccessibilityServiceInfo.FEEDBACK_SPOKEN;
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.when;
-import static org.mockito.MockitoAnnotations.initMocks;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
 
 public class AccessibilityServicesTest {
 
-    @Mock
-    AccessibilityManager mockAccessibilityManager;
-
-    @Mock
-    AccessibilityServiceInfo mockAccessibilityServiceInfo;
+    private final AccessibilityManager accessibilityManager = mock(AccessibilityManager.class);
+    private final CaptionManager captionManager = mock(CaptionManager.class);
+    private final AccessibilityServices accessibilityServices = new AccessibilityServices(accessibilityManager, captionManager);
 
-    @Mock
-    CaptionManager mockCaptionManager;
+    @Test
+    public void givenEnabledServicesListForSpokenFeedbackIsNotEmpty_thenReportsSpokenFeedbackIsEnabled() {
+        AccessibilityServiceInfo anyServiceInfo = accessibilityServiceInfoWithId("any");
+        given(accessibilityManager.getEnabledAccessibilityServiceList(FEEDBACK_SPOKEN))
+                .willReturn(Collections.singletonList(anyServiceInfo));
 
-    private AccessibilityServices accessibilityServices;
+        boolean spokenFeedbackEnabled = accessibilityServices.isSpokenFeedbackEnabled();
 
-    @Before
-    public void setUp() {
-        initMocks(this);
-        accessibilityServices = new AccessibilityServices(mockAccessibilityManager, mockCaptionManager);
+        assertThat(spokenFeedbackEnabled).isTrue();
     }
 
     @Test
-    public void whenAccessibilityServiceInfoListIsNotEmpty_thenSpokenFeedbackIsEnabled() {
-        List<AccessibilityServiceInfo> serviceInfoList = Collections.singletonList(mockAccessibilityServiceInfo);
-        when(mockAccessibilityManager.getEnabledAccessibilityServiceList(AccessibilityServiceInfo.FEEDBACK_SPOKEN))
-                .thenReturn(serviceInfoList);
+    public void givenEnabledServicesListForSpokenFeedbackIsEmpty_thenReportsSpokenFeedbackIsDisabled() {
```

Comment:
renamed this test, it's functionally the same. swapped when/thenReturns for given/willReturn